docs/content/usage.md

```diff
@@ -182,6 +182,37 @@ $ gomplate -c .=http://xkcd.com/info.0.json -i '<a href="{{ .img }}">{{ .title }
 <a href="https://imgs.xkcd.com/comics/diploma_legal_notes.png">Diploma Legal Notes</a>
 ```
 
+### `--missing-key`
+
+Changes behavior if key not exist in the datasource.
+
+Available values:
+- `error` (default) - Exit with error if key not exist
+- `default`, `invalid` or `zero` - prints `<no value>` if key not exist
```

Comment:
Because there _might_ be situations where a non-`nil` zero value is present, I think this should be split up. And, we can take the wording from the Go docs too for consistency:

```suggestion
- `error` (default): Execution stops immediately with an error.
- `default` or `invalid`: Do nothing and continue execution. If printed, the result is the string `"<no value>"`.
- `zero`: The operation returns the zero value for the element (which may be `nil`, in which case the string `"<no value>"` is printed).
```